feed/serializers/serializer.py

```diff
@@ -55,9 +55,10 @@ def _create_feed_generator(self) -> FeedGenerator:
         fg.register_extension("arxiv", ArxivExtension, ArxivEntryExtension)
 
         # Populate the feed
-        fg.id(self.link)
+        link=self.link+categories
```

Comment:
Is this right if categories is None ?

Reply:
theres always a category, removed optional type in function